pyDeltaRCM/default.yml

```diff
@@ -99,7 +99,7 @@ save_velocity_grids:
   default: False
 save_dt:
   type: 'int'
-  default: 50
+  default: 86400
```

Comment:
Yeah, this is a good point for some discussion I think. The [BMI suggests not using years](https://bmi-spec.readthedocs.io/en/latest/#get-time-units) because it is imprecisely defined.

Since the natural SI unit of other quantities is generally in seconds, using anything other than seconds for the timestep generally requires converting between time-units. This is doable, for sure, but it is a potential (likely) source of bugs and errors.

Since the max value for a `int64` is (I think) approximately 1 x 10**18 seconds, or 2 x 10**11 yrs (over 200 billion years), I think it is _safe_ to use seconds as the unit. 

Maybe we could provide convenience function to get the current time formatted as something other than seconds, but I think seconds is the safest choice.